elytron/src/test/resources/org/wildfly/extension/elytron/credential-security-factories.xml

```diff
@@ -16,6 +16,7 @@
                                    server="false"
                                    obtain-kerberos-ticket="true"
                                    debug="true"
+                                   mechanisms="KRB5 KRB5LEGACY"
```

Comment:
And can I configure KRB5+SPNEGO with:
* mechanisms="KRB5 SPNEGO"
or
* mechanism-oids="1.2.840.113554.1.2.2 1.3.6.1.5.5.2"
?